debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/pointer/helper/J9IndexableObjectHelper.java

```diff
@@ -142,6 +142,36 @@ public static long discontiguousHeaderSize()
 		return J9IndexableObjectDiscontiguous.SIZEOF;
 	}
 
+	/**
+	 * @param objPointer the contiguous array object who's dataAddr field we are accessing
+	 * @throws CorruptDataException 
+	 */
+	public static VoidPointer getDataAddrForContiguous(J9IndexableObjectPointer objPointer) throws CorruptDataException, NoSuchFieldException
```

Comment:
You should document the possibility of `NoSuchFieldException` (although I'm still deciding whether it should be handled here).